.github/workflows/update-image-digest.yml

```
@@ -6,7 +6,8 @@ on:
       - update-image-digest
 
 concurrency:
-  group: deploy-${{ join(github.event.client_payload.images, ',') }}
+  group: update-image-digest
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)